data_download_scripts/parse_dem.py

```diff
@@ -0,0 +1,15 @@
+from pathlib import Path
+import xarray as xr
+
+
+folder = "C:/STEMMUS_SCOPE_data/global/dem"
+fname = "Copernicus_DSM_30_N61_00_E024_00_DEM.tif"
+
+lat = 61.8474
+lon = 24.2948
+file = Path(folder) / fname
+
+ds = xr.open_dataarray(file, engine="rasterio")
+
+elevation = ds.sel(x=lon, y=lat, method="nearest")
+print(elevation.values[0])
```

Comment:
Like other notebook examples `parse_*.ipynb`, let's move these to a notebook `parse_dem.ipynb` instead of a python script.